src/content/pages/thank-you.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Thank you!
+---
+
+import DownloadLink from 'components/DownloadLink';
+import Panel from 'components/Panel';
+
+<Panel>
+
+Thank you so much for buying Mic Drop! ❤️ Your purchase supports independent software developers and helps fund new features. 
+
+## Getting your license
+
+You should receive a receipt and a license key via email momentarily. If it doesn't come through (check your spam folders just in case!), please [drop us a line](mailto:help@getmicdrop.com) and we'll send it over manually. 
```

Comment:
```suggestion
You should receive a receipt and a license key via email momentarily. If it doesn't come through (check your spam folders just in case!), please [drop us a line](mailto:help@getmicdrop.com) and we'll get you sorted. 
```